hazelcast/src/main/java/com/hazelcast/spi/impl/NodeEngineImpl.java

```diff
@@ -425,11 +425,11 @@ public void reset() {
 
     public void shutdown(final boolean terminate) {
         logger.finest("Shutting down services...");
+        operationService.shutdown();
```

Comment:
shutdown ordering changes like this are always somewhat frightening. for example the wait-notify service uses operation-service during shutdown. I assume you know what you are doing ;-)